tests/acceptance/tests/Visual/VisualFinalOrderPage.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@fixtures/AcceptanceTest';
+import path from 'path';
+
+test('Visual: Final order page in the Storefront.', { tag: '@Visual' }, async ({
```

Comment:
We can use same naming as our page objects to make it easy to follow. Even we can separate Admin and Storefront pages in a separate folders if possible.
```suggestion
test('Visual: Storefront:Checkout-Finish page in the Storefront.', { tag: '@Visual' }, async ({
```

Reply:
We have separated the page objects, but not the actual tests, right? Please let me know if you still think that separating visual tests is advised and how to do it.